apps/app/src/pages/Build/BuildStatusInfoCard.js

```diff
@@ -54,8 +80,10 @@ function getStatusCardParams({ type, status, screenshotDiffs }) {
             title: "Build in error",
             body: (
               <>
-                An error happened. Try again or contact us on{" "}
-                <Link href="https://discord.gg/pK79sv85Vg">Discord.</Link>
+                An error occurred while creating the build. If you are having
```

Comment:
This is not exact, this is not at the build creation, else it will not be visible on the UI.

> The build failed to be processed. If you are having trouble with this issue, reach us on <Link href="https://discord.gg/pK79sv85Vg">Discord</Link>.